editor/editor_autoload_settings.cpp

```diff
@@ -890,3 +903,14 @@ EditorAutoloadSettings::~EditorAutoloadSettings() {
 		}
 	}
 }
+
+void EditorAutoloadSettings::_chosen_path(const String &p_text) {
```

Comment:
I know it has been called `_chosen` originally, but I would find a new name for this function, so that it has a verb.

Reply:
`_select_path` or `_choose_path`?

Reply:
`_set_autoload_path` seems to be a good name.

Reply:
Yes, it may be good to work with the name of the variable that is holding LineEdit.

Reply:
Check now.

Reply:
LGTM, thanks